### DISCUSSION TITLE
Midi player loop

### DISCUSSION THREAD

**Discussion by BigHonkinDaddy** [powertab/powertabeditor#466] [Ideas]
I'd like to be able to set "start" and "end" markers in the midi player.  I typically use the midi play back to practice sections of a new song.  Constantly having to reset/replay with the pc mouse is cumbersome when I want to keep my hands on the guitar.

**Comment by cameronwhite** [comment]
👍 Definitely agree this would be a great feature to add.  I think this is generally covered by bugs #117 and #359 so I'll close this in favour of those
